_help/index.md

```diff
@@ -9,7 +9,15 @@ type: markdown
 toc: false
 ---
 
-{% include section-index.html docs=site.help %}
+Here are some resources to help you deploy, configure and use Istio.
+
+- [Frequently Asked Questions]({{home}}/help/faq/). You ask 'em, we'll answer 'em.
+
+- [Glossary]({{home}}/help/glossary.html). A glossary of common Istio terms.
+
+- [Reporting Bugs]({{home}}/help/bugs.html). What to do when you find a bug.
+
+- [Troubleshooting Guide]({{home}}/help/troubleshooting.html). Practical advice on practical problems with Istio.
```

Comment:
"Practical advice on practical problems" is fairly redundant, is this what we want? (I think it's fine, just double checking.)